google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/ui/CloudAttachDialog.java

```diff
@@ -397,7 +389,7 @@ private boolean stash() {
       GitHandlerUtil.doSynchronously(
           handler, GitBundle.getString("stashing.title"), handler.printableCommandLine());
     } finally {
-      DvcsUtil.workingTreeChangeFinished(project, token);
+      token.finish();
```

Comment:
mind explaining this change just so I understand the reasons / differences?

Reply:
Sure, it's an easy one - IDEA looks through the docs and sources and suggests removing deprecated code - this method has been deprecated and recommends calling token.finish() instead in Javadoc and sources. It actually calls `token.finish()` in deprecated `workingTreeChangeFinished`. I know it's not a change aligned with this PR, just minor clean up in the same code, what do you think?

Reply:
I'm fine with it. Probably worth testing out just to be certain they didn't break anything.

Reply:
ah nm. looks like the deprecated method was just doing this anyway

Reply:
which you did already say in your original post if I had read it more carefully :)

Reply:
My fault, TLDR applies! :)